### DISCUSSION TITLE
VSCode Not Invoked When Creating GitHub Codespace

### DISCUSSION THREAD

**Discussion by conradwt** [community/community#64699] [Codespaces]
### Select Topic Area

Bug

### Body

I'm seeing a similar issue.  For example,

1. Create GitHub Codespace
2. I see "Launching Visual Studio Code" message

![Screenshot 2023-07-30 at 11 21 01 PM](https://github.com/microsoft/vscode/assets/20219/f9e190ca-4a34-45df-838b-640399ccefbe)

Finally, I have VSCode Insiders 1.82.0-insider installed on macOS 13.5.1 (Ventura).

**Comment by arafajs** [comment]
- Make sure that you have the latest version of the GitHub Codespaces extension installed. You can check this by opening Visual Studio Code and going to the Extensions view. The extension should be called "GitHub Codespaces" and be published by Microsoft.
- Make sure that you are using the stable build of Visual Studio Code. The Insiders release is a preview release and may not be compatible with GitHub Codespaces.
- Check your network connection. If your network is blocking connections to GitHub, you may not be able to create a codespace.
- If you are using a corporate network, your IT department may have blocked access to GitHub Codespaces. You can contact your IT department for more information.
- If you have checked all of these things and VSCode is still not being invoked, you can try the following:
- Restart Visual Studio Code.
- Clear the Visual Studio Code cache. To do this, open a command prompt and navigate to the Visual Studio Code installation directory. Then, run the following command:
- `code --clear-cache`
- Reinstall the GitHub Codespaces extension.
